cmd/csi-node-driver-registrar/main.go

```diff
@@ -46,6 +46,7 @@ const (
 var (
 	connectionTimeout       = flag.Duration("connection-timeout", 0, "The --connection-timeout flag is deprecated")
 	csiAddress              = flag.String("csi-address", "/run/csi/socket", "Path of the CSI driver socket that the node-driver-registrar will connect to.")
+	pluginRegistrationPath  = flag.String("plugin-registration-path", "/registration", "Path of the registration socket on the Kubernetes host machine.")
```

Comment:
It's not "not "path of the registration socket" - it directory name. And it's not anything on Kubernetes host machine, but path where the plugin registration directory is exposed in the registrar container,

Reply:
ah - we're using this not in a sidecar container, but running the registrar directly as a `systemd` process on a host machine (Fargate task)

Reply:
I can update the description as you guys see fit.

Reply:
What about "Path to Kubernetes plugin registration directory"?

Reply:
@prithviramesh, can you please update the description?

Reply:
I can, what you suggested SGTM. Will push up another rev today